### ISSUE TITLE
Zone update crash when invalid characters are present, with no front-end alert

### ISSUE THREAD

**Issue by iget-master** [plesk/ext-route53#13]
The front-end lacks of a whitespace/tab characters validation when inserting DNS entries.
If I insert 20 entries but only one have an invalid character, the zone is not updated on Route53 and this message is logged:

```
Execution of /opt/psa/admin/plib/modules/route53/scripts/route53.php failed with exit code 255 and the output:
Failed zone update snackout.com.br.: Invalid Resource Record: FATAL problem: UnsupportedCharacter (Value contains unsupported characters) encountered with '	'
```

Maybe this script lacks a whitespace trim to avoid this kind of issue.

**Comment by iget-master** [comment]
Could someone help here?!

**Comment by ayamshanov** [comment]
One of our customers reported the same problem. After a small investigate, we found the problem with TXT record. In this case, an unsupported symbol was a space at the start of TXT value. The issue was fixed by removing this "space" symbol.

**Comment by vlikhtanskiy** [comment]
Fixed in version 2.5
